tempest/api/workloadmgr/restore/test_exclude_cinder_volume.py

```diff
@@ -0,0 +1,266 @@
+import json
+import os
+import sys
+import time
+import paramiko
+
+import yaml
+from oslo_log import log as logging
+
+from tempest import command_argument_string
+from tempest import config
+from tempest import reporting
+from tempest import tvaultconf
+from tempest.api.workloadmgr import base
+from tempest.lib import decorators
+from tempest.util import cli_parser
+from tempest.util import query_data
+
+sys.path.append(os.getcwd())
+
+LOG = logging.getLogger(__name__)
+CONF = config.CONF
+
+
+class WorkloadTest(base.BaseWorkloadmgrTest):
+
+    credentials = ['primary']
+
+    @classmethod
+    def setup_clients(cls):
+        super(WorkloadTest, cls).setup_clients()
+
+
+    def assign_floating_ips(self, vm_id, cleanup):
+        fip = self.get_floating_ips()
+        LOG.debug("\nAvailable floating ips are {}: \n".format(fip))
+        self.set_floating_ip(str(fip[0]), vm_id, floatingip_cleanup=cleanup)
+        return(fip[0])
+
+
+    def data_ops_on_mount_point(self, flo_ip, mount_point, file_count):
+        global volumes
+        ssh = self.SshRemoteMachineConnectionWithRSAKey(str(flo_ip))
+        stdin, stdout, stderr = ssh.exec_command(
```

Comment:
Use already existing method install_qemu() from base.py instead of repeating it here.